site/src/content/docs/commands/zarf_package_pull.md

```diff
@@ -31,8 +31,9 @@ $ zarf package pull oci://ghcr.io/defenseunicorns/packages/dos-games:1.0.0 -a sk
 ### Options
 
 ```
-  -h, --help                      help for pull
-  -o, --output-directory string   Specify the output directory for the pulled Zarf package
+  -h, --help                              help for pull
+  -o, --output-directory string           Specify the output directory for the pulled Zarf package
+      --shasum zarf dev sha256sum <url>   Shasum of the package to pull. Required if pulling a https package. A shasum can be retrieved using zarf dev sha256sum <url>
```

Comment:
The description here looks weird. `zarf dev sha256sum <url>` instead of `string`?

Reply:
ohh good catch. Cobra must react weirdly with `` in help text. Switching to ''